spec/factories/votes.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :vote do
+    user
+    review
+    vote_number [-1, 1].sample
```

Comment:
Instead of letting fate decide what vote you have, you can have traits in your factory so you can create upvotes and downvotes

Reply:
We didn't add this in, this was what Al and Bobby's group are doing for their pull request. This pull request is only for the administrative rights to delete reviews and art labels.

Reply:
Just a suggestion!